src/lib/application/application.factory.ts

```diff
@@ -29,7 +30,7 @@ export function main(options: ApplicationOptions): Rule {
   return mergeWith(generate(options, path));
 }
 
-function transform(options: ApplicationOptions): ApplicationOptions {
+export function transform(options: ApplicationOptions): ApplicationOptions {
```

Comment:
I exported this internal `transform` function because I didn't find any other way to write the unit test that ensures that `express` is used by default (and so, make this feat backward-compatible)